### PR TITLE
Feature/add serializer class in model observer

### PR DESCRIPTION
**Pull request by lautarodapin**
I added a logic to pass a kwargs to the ``@model_observer`` decorator, this way it handles automatically the serialization

### CODE REVIEW

**Comment by hishnash** [comment]
@lautarodapin sweet thank for adding this i will give this a review later today/tomorrow morning

**Comment by lautarodapin** [comment]
@hishnash remember review it D:

File: djangochannelsrestframework/observer/model_observer.py [hunk 1]
```diff
@@ -184,6 +187,8 @@ def serialize(self, instance, action, **kwargs) -> Dict[str, Any]:
         message_body = {}
         if self._serializer:
             message_body = self._serializer(self, instance, action, **kwargs)
+        elif self._serializer_class:
+            message_body = self._serializer_class(instance).data
```
Comment (hishnash) [inline djangochannelsrestframework/observer/model_observer.py:191]: could you add a test that checks this works as expected on a delete action.

**Review by hishnash — APPROVED** [review]

File: djangochannelsrestframework/observer/model_observer.py (hunk 1 above)
Comment (lautarodapin) [reply djangochannelsrestframework/observer/model_observer.py:191]: ok!

**Review by hishnash — APPROVED** [review]

**Comment by hishnash** [comment]
This is sweet thankyou for adding it !

**Pull request merged**